Config handler fixes

antonyagustine opened the pull request:
- Lodash dependency dependency added in package manager

netrajpatel reviewed (commented):
@antonyagustine Though the complexity wise both ways looks simillar, using lodash.isEmpty() makes our code more readable as well as handles more cases of array as well as object with boolean return type rather than integer.
Also, we should use uniform code structure everywhere to reduce the further confusions, So I think, you should use the lodash isEmpty which is better maintainable in future as well.
I'm open for any further discussions, let me know.

antonyagustine commented:
> @antonyagustine Though the complexity wise both ways looks simillar, using lodash.isEmpty() makes our code more readable as well as handles more cases of array as well as object with boolean return type rather than integer. Also, we should use uniform code structure everywhere to reduce the further confusions, So I think, you should use the lodash isEmpty which is better maintainable in future as well. I'm open for any further discussions, let me know.

@netrajpatel I've reverted the code back to lodash. @shafeeqd959 As we discussed I've added lodash dependency in package manager.

netrajpatel commented on packages/contentstack-utilities/src/logger.ts:
```diff
@@ -1,5 +1,5 @@
-import * as winston from 'winston';
-import config from './config-handler';
+import winston from 'winston';
+// import config from './config-handler';
```
Remove unused/commented code
